rules/retrieve.smk

```diff
@@ -56,6 +56,24 @@ if config["enable"]["retrieve"] and config["enable"].get("retrieve_databundle",
         script:
             "../scripts/retrieve_eurostat_data.py"
 
+    rule retrieve_jrc_idees:
+        output:
+            directory("data/bundle/jrc-idees-2021"),
```

Comment:
```suggestion
            directory("data/jrc-idees-2021"),
```
For similar "outsourcing" from the data bundle, I have moved the files out of the `data/bundle` directory.